src/cf-tables.less

```diff
@@ -370,11 +380,24 @@
 }
 
 .table__sortable {
+    button.sortable {
+        width: 100%;
+        height: 100%;
+        padding: inherit;
+        margin: inherit;
+        border: none;
+        background: none;
+        font-family: inherit;
+        font-weight: inherit;
+        outline: none;
+        text-align: left;
+    }
```

Comment:
> Here's an idea: 

I like this idea! I had thought of something similar - afterall, `button` is extremely useful as an interactive element for apps, and including an easy way to have an unstyled `button` will come in handy for us in the future, I think.